### PR TITLE
Release v0.5.3

### PR DESCRIPTION
**Pull request by phil-opp**
Publishes `v0.5.3` of `volatile` and `volatile-macro`.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
-- Add `VolatileRef::borrow` and `VolatileRef::borrow_mut`
+* Add `#[derive(Volatile)]` for easy, access-limited field-based access to structs by @mkroening in https://github.com/rust-osdev/volatile/pull/49
```
Comment (mkroening) [inline Changelog.md]:
```suggestion
* Add `#[derive(VolatileFieldAccess)]` for easy, access-limited field-based access to structs by @mkroening in https://github.com/rust-osdev/volatile/pull/49
```

**Review by mkroening — COMMENTED** [review]
It would be great if we could squeeze https://github.com/rust-osdev/volatile/pull/55 in before this. Sorry for the noise. :)

**Comment by phil-opp** [comment]
I created a test `v0.5.3-rc.1` release using `cargo release rc --workspace`. It looks like it handled the version bump for both crates and the dependency update correctly.

File: Changelog.md (hunk 1 above)
Comment (phil-opp) [reply Changelog.md]: Thanks!

**Comment by phil-opp** [comment]
Published a `v0.5.3-rc.2` release. The derive macro now shows up on docs.rs: https://docs.rs/volatile/0.5.3-rc.2/volatile/index.html

@mkroening Ready for publish from your side?

**Comment by mkroening** [comment]
> Ready for publish from your side?

Yes, thank you! :)

**Pull request merged**